### PR TITLE
state persistance and hardcode pixel transparency value

### PR DESCRIPTION
**Pull request by LittleChimera**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ module.exports = {
       NODE_ENV: "development",
       CANVAS_BACKEND_CANISTER_ID: canisters["canvas_backend"],
       II_URL: isDevelopment
-        ? "http://localhost:8000?canisterId=rrkah-fqaaa-aaaaa-aaaaq-cai#authorize"
+        ? "http://localhost:8000?canisterId=r7inp-6aaaa-aaaaa-aaabq-cai#authorize"
```
Comment (LittleChimera) [inline webpack.config.js]:
```suggestion
        ? "http://localhost:8000?canisterId=rrkah-fqaaa-aaaaa-aaaaq-cai#authorize"
```

File: src/canvas_backend/src/api.rs [hunk 2]
```diff
@@ -31,6 +36,12 @@ pub struct Color {
     pub a: u8,
 }
 
+#[candid_method(query)]
+#[query]
+pub fn time_left() -> u64 {
+    storage::get::<EditsState>().time_left()
+}
+
```
Comment (LittleChimera) [inline src/canvas_backend/src/api.rs:44]: this is not generated in candid for some reason...

File: src/canvas_assets/src/Submit.tsx [hunk 3]
```diff
@@ -107,7 +107,7 @@ function Submit(props: { handleDrop: any; renderCanvas2: any }) {
             cancelLabel="cancel"
             isPrimaryActionDisabled={!isAuthenticated}
             width="size-500"
-            isHidden={!isConfirmDisabled && isAuthenticated}
+            isHidden={!isConfirmDisabled && !isAuthenticated}
```
Comment (LittleChimera) [inline src/canvas_assets/src/Submit.tsx]: @krpeacock submit window was not opening for me. I think this should be correct. please double-check :)

Comment (krpeacock) [reply src/canvas_assets/src/Submit.tsx]: Oh, I didn't test that flow adequately. I'll verify the logic

File: src/canvas_backend/src/api.rs (hunk 2 above)
Comment (krpeacock) [reply src/canvas_backend/src/api.rs:44]: I've been manually updating the candid by calling the tmp hack from dfx

**Pull request merged**